static/bidder-info/aduptech.yaml

```diff
@@ -0,0 +1,27 @@
+endpoint: "https://rtb.d.adup-tech.com/rtb/bid"
+extra_info: "{\"target_currency\": \"EUR\"}"
+geoscope:
+  - EEA
+  - CHE
+  - GBR
+maintainer:
+  email: "support@adup-tech.com"
+gvlVendorID: 647
+openrtb:
+  version: 2.6
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - native
+userSync:
+  redirect:
```

Comment:
@danygielow The usersync URL returned by the /cookie_sync for `aduptech` is currently throwing 404. Can you please check if the URLs added for userSync are correct?

Reply:
Fixed sync Endpoint.